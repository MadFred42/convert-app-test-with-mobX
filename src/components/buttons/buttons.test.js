import React from 'react';
import Buttons from './';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing <Buttons/>', () => {
    it('Buttons have rendered correctly', () => {
        const buttons = shallow(<Buttons />);
        expect(toJson(buttons)).toMatchSnapshot();
    });
})