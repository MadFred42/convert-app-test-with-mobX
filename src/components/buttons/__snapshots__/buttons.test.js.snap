// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Buttons/> Buttons have rendered correctly 1`] = `
<div
  className="d-flex justify-content-center"
>
  <Link
    to="/"
  >
    <button
      className="btn btn-outline-primary"
    >
      Converter
    </button>
  </Link>
  <Link
    to="/currency"
  >
    <button
      className="btn btn-outline-primary"
    >
      Exchange rates
    </button>
  </Link>
</div>
`;
